packages/sync/src/class-settings.php

```diff
@@ -204,7 +204,7 @@ public static function update_settings( $new_settings ) {
 			}
 
 			// If we set the disabled option to true, clear the queues.
-			if ( ( 'disable' === $setting || 'network_disable' === $setting ) && ! ! $value ) {
+			if ( ( 'disable' === $setting || 'network_disable' === $setting ) && (bool) $value ) {
```

Comment:
We could drop the `(bool)` entirely, PHP implicitly casts to bool in this situation anyway.